python-avd/pyavd/_eos_designs/eos_designs_facts/vlans.py

```diff
@@ -86,9 +86,9 @@ def _local_endpoint_vlans_and_trunk_groups(self: EosDesignsFacts) -> tuple[set,
         for connected_endpoints_key in self.shared_utils.connected_endpoints_keys:
             connected_endpoints = get(self._hostvars, connected_endpoints_key["key"], default=[])
             for connected_endpoint in connected_endpoints:
-                for adapter in connected_endpoint.get("adapters", []):
+                for index, adapter in enumerate(connected_endpoint.get("adapters", [])):
                     adapter_settings = self.shared_utils.get_merged_adapter_settings(
-                        adapter, context=f"{connected_endpoints_key['key']}[{connected_endpoint['name']}].adapters"
+                        adapter, context=f"{connected_endpoints_key['key']}[{connected_endpoint['name']}].adapters[{index}]"
```

Comment:
```suggestion
                        adapter, context=f"{connected_endpoints_key['key']}[name={connected_endpoint['name']}].adapters[{index}]"
```

Reply:
thats not what we have in the other places where the message is so would need to change it there as well